app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::GreetingsController < ApplicationController
+  def index
+    @greeting = Message.order('RANDOM()').first
+    render json: { message: @greeting.content }
+  end
+end
```

Comment:
- [ ] When navigate to the `/api/greeting` routes I got this error :point_down: (see image). Kindly make sure the greeting route is available and return a random greeting. 
![Screenshot from 2023-11-28 12-36-51](https://github.com/BranBayou/hello-rails-react/assets/91142432/96f0f094-ed66-4bbe-aa12-325805ce190c)